notes/notes.org

```diff
@@ -779,9 +779,9 @@ stored bits in the significand and a (stored) exponent range of \([- 7, 8]\)?
 #+LATEX: \begin{showlater}
 - Can go way smaller using the /special exponent/ (turns off the leading one)
 - Assume that the special exponent is \(- 7\).
-- So: \((0.001)_2 \cdot 2^{-7}\) (with all four digits stored).
+- So: \((0.0001)_2 \cdot 2^{-7}\) (with all four digits stored).
```

Comment:
Thanks for bringing this to my attention. Upon re-reading, I no longer like any part of it. I'll just rewrite it from scratch in a different PR. (I'll tag you to review.)

Reply:
How do you like #10?